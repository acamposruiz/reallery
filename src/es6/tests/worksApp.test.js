import React from 'react';
import WorksApp from '../worksApp';
import renderer from 'react-test-renderer';
import state from './mocks/state';

jest.mock('../projectButton', () => { return 'ProjectButton'; });
jest.mock('../projectGallery', () => { return 'ProjectGallery'; });
jest.mock('react-multimedia-gallery', () => { return 'Gallery'; });

test('WorksApp', () => {
  const component = renderer.create(
    <WorksApp meta={state.meta} model={state.projects}/>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});