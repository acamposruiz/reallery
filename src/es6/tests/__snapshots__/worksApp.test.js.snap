// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorksApp 1`] = `
<div
  className="container home no-mobile"
>
  <header
    className="container-header"
  >
    <h1
      className="title-header"
    >
      <a
        className="main-title"
        href="/#/en"
      >
        Michelangelo Buonarroti
      </a>
    </h1>
    <section
      className="menu menu-header"
    >
       
       
    </section>
  </header>
  <section
    className="menu menu-content"
  >
     
    <ProjectButton
      active={false}
      color="F0B67F"
      icon={
        Object {
          "family": "FontAwesome",
          "icon": "faPaintBrush",
        }
      }
      key2="paintings"
      lng="en"
      name="Paintings"
      publish={true}
      state=""
      type={undefined}
      view="home"
    />
    <ProjectButton
      active={false}
      color="D6D1B1"
      icon={
        Object {
          "family": "FontAwesome",
          "icon": "faCube",
        }
      }
      key2="sculptures"
      lng="en"
      name="Sculptures"
      publish={true}
      state=""
      type={undefined}
      view="home"
    />
    <ProjectButton
      active={false}
      color="C7EFCF"
      icon={
        Object {
          "family": "FontAwesome",
          "icon": "faUniversity",
        }
      }
      key2="architecture"
      lng="en"
      name="Architecture"
      publish={true}
      state=""
      type={undefined}
      view="home"
    />
     
  </section>
  <section
    className="gallery"
  >
     
    <ProjectGallery
      lng="en"
      project={null}
    />
     
  </section>
  <footer
    className="contact"
  >
    <a
      href="https://github.com/acamposruiz/reallery"
      target="_blank"
    >
      <span
        className="mail-data"
      >
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": undefined,
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m15.8 27.1q0 0.9-0.3 1.9t-1 1.7-1.6 0.7-1.6-0.7-1-1.7-0.2-1.9 0.3-1.8 0.9-1.7 1.6-0.7 1.6 0.7 1 1.7 0.3 1.8z m14.3 0q0 0.9-0.3 1.9t-1 1.7-1.6 0.7-1.6-0.7-1-1.7-0.2-1.9 0.2-1.8 1-1.7 1.6-0.7 1.6 0.7 1 1.7 0.3 1.8z m3.5 0q0-2.6-1.5-4.5t-4.2-1.9q-0.9 0-4.3 0.5-1.6 0.2-3.5 0.2t-3.5-0.2q-3.4-0.5-4.4-0.5-2.6 0-4.2 1.9t-1.5 4.5q0 2 0.7 3.5t1.8 2.3 2.7 1.3 3.2 0.7 3.3 0.1h3.7q1.9 0 3.4-0.1t3.1-0.7 2.7-1.3 1.8-2.3 0.7-3.5z m5-3.9q0 4.6-1.3 7.4-0.9 1.7-2.4 3t-3.1 1.9-3.8 1.1-3.8 0.4-3.8 0.1q-1.7 0-3.1 0t-3.3-0.3-3.4-0.7-3.1-1.1-2.7-1.8-1.9-2.6q-1.4-2.7-1.4-7.4 0-5.3 3-8.8-0.6-1.9-0.6-3.8 0-2.6 1.2-4.9 2.4 0 4.2 0.9t4.2 2.8q3.3-0.8 6.9-0.8 3.3 0 6.3 0.7 2.3-1.8 4.2-2.7t4.2-0.9q1.1 2.3 1.1 4.9 0 1.9-0.6 3.7 3 3.6 3 8.9z"
            />
          </g>
        </svg>
        <span
          className="data"
        >
          View on github
        </span>
      </span>
    </a>
    <a
      href="mailto:mbuonarroti@reallery.com"
    >
      <span
        className="mail-data"
      >
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": undefined,
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m40 15.8v17.8q0 1.4-1 2.5t-2.6 1h-32.8q-1.5 0-2.6-1t-1-2.5v-17.8q1 1.1 2.3 2 8 5.5 11 7.7 1.3 0.9 2.1 1.5t2.1 1 2.5 0.6h0q1.2 0 2.5-0.6t2.1-1 2.1-1.5q3.7-2.8 11.1-7.7 1.2-0.9 2.2-1.9z m0-6.5q0 1.7-1.1 3.4t-2.7 2.7q-8.4 5.8-10.5 7.3-0.2 0.1-0.9 0.6t-1.2 0.9-1.2 0.7-1.3 0.6-1.1 0.2h0q-0.5 0-1.1-0.2t-1.3-0.6-1.2-0.7-1.2-0.9-0.9-0.6q-2.1-1.5-5.9-4.1t-4.6-3.2q-1.3-0.9-2.6-2.6t-1.2-3q0-1.8 0.9-2.9t2.7-1.2h32.8q1.5 0 2.5 1.1t1.1 2.5z"
            />
          </g>
        </svg>
        <span
          className="data"
        >
          mbuonarroti@reallery.com
        </span>
      </span>
    </a>
           
    <a
      href="tel:555-555-555"
    >
      <span
        className="mail-data"
      >
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": undefined,
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m35.9 27.7q0 0.6-0.2 1.6t-0.5 1.5q-0.4 1.1-2.7 2.3-2.1 1.2-4.1 1.2-0.6 0-1.2-0.1t-1.3-0.3-1.1-0.3-1.2-0.5-1.1-0.4q-2.2-0.7-3.9-1.8-2.8-1.8-5.9-4.8t-4.8-5.9q-1.1-1.7-1.9-3.9 0-0.2-0.4-1.1t-0.4-1.3-0.3-1-0.3-1.3-0.1-1.2q0-2 1.1-4.1 1.3-2.3 2.4-2.8 0.6-0.2 1.5-0.4t1.6-0.2q0.3 0 0.5 0 0.4 0.2 1.2 1.7 0.2 0.4 0.6 1.2t0.8 1.4 0.7 1.2q0.1 0.1 0.4 0.6t0.5 0.8 0.1 0.6q0 0.5-0.6 1.1t-1.4 1.3-1.4 1.1-0.6 1.1q0 0.2 0.1 0.5t0.2 0.4 0.3 0.6 0.3 0.4q1.7 3.1 3.8 5.2t5.3 3.9q0 0 0.4 0.3t0.6 0.3 0.4 0.2 0.5 0.1q0.4 0 1-0.6t1.2-1.4 1.3-1.4 1.1-0.6q0.3 0 0.6 0.1t0.8 0.5 0.6 0.4q0.5 0.3 1.1 0.7t1.5 0.8 1.2 0.6q1.5 0.8 1.7 1.2 0 0.2 0 0.5z"
            />
          </g>
        </svg>
        <span
          className="data"
        >
          555-555-555
        </span>
      </span>
    </a>
  </footer>
</div>
`;
